File: leohoot/src/model/answer.ts
export class Answer {
  private _answer: string;
  private _isCorrect: boolean;

  constructor(answer: string, isCorrect: boolean) {
    this._answer = answer;
    this._isCorrect = isCorrect;
  }

  get answer(): string {
    return this._answer;
  }

  set answer(answer: string) {
    this._answer = answer;
  }

  get isCorrect(): boolean {
    return this._isCorrect;
  }

  set isCorrect(isCorrect: boolean) {
    this._isCorrect = isCorrect;
  }
}

File: leohoot-backend/Core/Entities/Answer.cs
using System.ComponentModel.DataAnnotations;
using Base.Core.Entities;

namespace Core.Entities;
public class Answer: EntityObject
{
    [MaxLength(256)]
    public required string AnswerText { get; set; }
    public bool IsCorrect { get; set; }
    public int QuestionId { get; set; }
    public Question? Question { get; set; }
}